docs/source/user-guide/mission-modeler-guide/activity-types/effect-model.md

```diff
@@ -0,0 +1,128 @@
+# Effect Model
+Every activity type has an associated "effect model" that describes how that activity impacts mission resources. An effect model is a method on the activity type class annotated with `@EffectModel`; there can only be one such method, and by convention it is named `run` that accepts the top-level `Mission` model as a parameter. This method is invoked when the activity begins, paused when the activity waits a period of time, and resumes when that period of time passes. The activity's computed duration will be measured from the instant this method is entered, and extends either until the method returns or until all spawned children of the activity have completed -- whichever is longer.
+
+**Exceptions:**
+If an uncaught exception is thrown from an activity's effect model, the simulation will halt. No later activities will be performed, and simulation time will not proceed beyond the instant at which the exception occurred. As such, uncaught exceptions are essentially treated as fatal errors. We advise mission models to employ uncaught exceptions sparingly, as it deprives planners of information about the behavior of the spacecraft after the fault. (At least as of 2021-06-11, an uncaught exception will cause the simulation to abort without producing *any* results -- not even up to the point of failure.) On the other hand, uncaught exceptions may be useful to identify bugs in the mission model or situations that the mission model is not intended to simulate.
+
+**Actions:**
+An activity's effect model may wait for time to pass, spawn other activities, and affect spacecraft state. Spacecraft state is affected via the `Mission` model parameter, which depends on the details of the modeled mission systems.
+
+Actions related to the passage of simulation time are provided as static methods on the `merlin.framework.ModelActions` class:
+
+- `delay(duration)`: Delay the currently-running activity for the given duration. On resumption, it will observe effects caused by other activities over the intervening timespan.
+- `waitFor(activityId)`: Delay the currently-running activity until the activity with specified ID has completed. On resumption, it will observe effects caused by other activities over the intervening timespan.
+- `waitUntil(condition)`: Delay the currently-running activity until the provided `Condition` becomes true. On resumption, it will observe effects caused by other activities over the intervening timespan.
+<!-- - `waitForChildren()`: Delay the currently-running activity until all child activities have completed. On resumption, it will observe effects caused by other activities over the intervening timespan. -->
```

Comment:
Why is this and line 20 surround by HTML comments?

Reply:
Thanks! Good catch, just removed these as they were from an older version of Merlin.